rmgpy/exceptions.py

```diff
@@ -0,0 +1,250 @@
+    any reason. Pass a string describing the cause of the exceptional behavior.
+    """
+    pass
+
+class ChemkinError(Exception):
+    """
+    An exception class for exceptional behavior involving Chemkin files. Pass a
+    string describing the circumstances that caused the exceptional behavior.
+    """
+    pass
+
+class CollisionError(Exception):
+    pass
+
+class DatabaseError(Exception):
+    """
+    A exception that occurs when working with an RMG database. Pass a string
+    giving specifics about the exceptional behavior.
+    """
+    pass
+
+class ElementError(Exception):
+    """
+    An exception class for errors that occur while working with elements.
+    Pass a string describing the circumstances that caused the
+    exceptional behavior.
+    """
+    pass
+
+class ForbiddenStructureException(Exception):
+    """
+    Made a forbidden structure.
+    """
+    pass
+
+class ILPSolutionError(Exception):
+    """
+    An exception to be raised when solving an integer linear programming problem if a solution
+    could not be found or the solution is not valid. Can pass a string to indicate the reason
+    that the solution is invalid.
+    """
+    pass
+
+class ImplicitBenzeneError(Exception):
+    """
+    An exception class when encountering a group with too many implicit benzene
+    atoms. These groups are hard to create sample molecules and hard for users
+    to interpret. Pass a string describing the limitation.
+    """
+    pass
+
+class InchiException(Exception):
+    pass
+
+class InputError(Exception):
+    """
+    An exception raised when parsing an input file for RMG or a specific conformer. 
+    Pass a string describing the error.
+    """
+    pass
+
+class InvalidActionError(Exception):
+    """
+    An exception to be raised when an invalid action is encountered in a
+    reaction recipe.
+    """
+    pass
+
+class InvalidAdjacencyListError(Exception):
+    """
+    An exception used to indicate that an RMG-style adjacency list is invalid.
+    Pass a string describing the reason the adjacency list is invalid
+    """
+    pass
+
+class KekulizationError(Exception):
+    """
+    An exception to be raised when encountering an error while kekulizing an aromatic molecule.
+    Can pass a string to indicate the reason for failure.
+    """
+    pass
+
+class KineticsError(Exception):
+    """
+    An exception for problems with kinetics. Pass a string describing the problem.
+    """
+    pass
+
+class ModifiedStrongCollisionError(Exception): 
+    """
+    An exception raised when the modified strong collsion method is unsuccessful
+    for any reason. Pass a string describing the cause of the exceptional 
+    behavior.
+    """
+    pass
+
+class NegativeBarrierException(Exception):
+    """This Exception occurs when the energy barrier for a hindered Rotor is negative.
+    This can occur if the scan or fourier fit is poor. """
+    
+    pass
+
+class NetworkError(Exception): 
+    pass
```

Comment:
The API says:  
NetworkError | Raised when an error occurs while working with a pressure-dependent reaction network